Admin/ArticleAdmin.php

```diff
@@ -136,13 +136,15 @@ protected function configureFormFields(FormMapper $formMapper)
             ->with('General', array('class' => 'col-md-8'))
                 ->add('title', TextType::class, array(
                     'attr' => array('maxlength' => 255),
+                    'help' => $this->trans('help_article_title'),
                 ))
                 ->add('subtitle', TextType::class, array(
                     'required' => false,
                     'attr' => array('maxlength' => 255),
                 ))
                 ->add('abstract', TextareaType::class, array(
                     'required' => false,
+                    'help' => $this->trans('help_article_abstract'),
```

Comment:
There is no specific twig template to render this, it's rendered by admin form_fields. Besides it's done like this on PageBundle so I've got to admit, didn't seek much further :)